controllers/clusteroperator.go

```diff
@@ -31,6 +32,9 @@ const (
 	// ReasonEmpty is an empty StatusReason
 	ReasonEmpty StatusReason = ""
 
+	// ReasonEmpty is an empty StatusReason
+	ReasonExpected StatusReason = "AsExpected"
```

Comment:
The comment does not match the variable here.

Reply:
Fixed.